echipa.html

```diff
@@ -87,22 +87,22 @@
                   <div class="collapse navbar-collapse" id="min_navbar">
                      <ul class="nav navbar-nav navbar-right">
                         <li class="dropdown submenu">
-                           <a href="index.html" class="">Home</a>
+                           <a href="./index.html" class="">Acasa</a>
                         </li>
                          <li class="dropdown submenu">
-                           <a href="about.html" class="">About</a>
+                           <a href="./despre.html" class="">Despre</a>
```

Comment:
Same as above ☝️

Create a folder with this name (**despre**) and place inside this folder the corresponding HTML file (**despre.html**), but rename it to index.html.


We do this so that the URL will not include the .html extension at the end and will be more user friendly.